certified-connectors/Gratavid/apiDefinition.swagger.json

```diff
@@ -0,0 +1,542 @@
+            "required": true,
+            "type": "string",
+            "default": "microsoftFlow",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "event",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "webhookSubscribe",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "webookHookEvent",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "taskSent",
+              "reply",
+              "noteWatched",
+              "unsubscribe",
+              "stopText"
+            ],
+            "x-ms-enum-values": [
+              {
+                "displayName": "Gratavid task sent to a contact",
+                "value": "taskSent"
+              },
+              {
+                "displayName": "Contact replies to a Gratavid",
+                "value": "reply"
+              },
+              {
+                "displayName": "Contact watches a Gratavid all the way through",
+                "value": "noteWatched"
+              },
+              {
+                "displayName": "Contact unsubscribes from email after receiving Gratavid email",
+                "value": "unsubscribe"
+              },
+              {
+                "displayName": "Contact opt-out of text (replies 'STOP') after receiving Gratavid",
+                "value": "stopText"
+              }
+            ],
+            "x-ms-visibility": "important",
+            "description": "Select the event you want to trigger on",
+            "x-ms-summary": "Gratavid Event"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "webhookURL": {
+                  "type": "string",
+                  "description": "webhookURL",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": ""
+                }
+              },
+              "required": [
+                "webhookURL"
+              ]
+            }
+          }
+        ],
+        "x-ms-visibility": "important"
+      }
+    },
+    "/api/integrationsEndpoint": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "message": {
+                  "type": "string",
+                  "description": "message"
+                },
+                "taskId": {
+                  "type": "string",
+                  "description": "taskId"
+                },
+                "viewInGratavidLink": {
+                  "type": "string",
+                  "description": "viewInGratavidLink"
```

Comment:
@Wtrapp All the description values of properties should be free from grammatical and spelling mistakes.